enarxbot-triage

```diff
@@ -16,9 +16,11 @@ with open(os.environ["GITHUB_EVENT_PATH"]) as f:
 if event["action"] not in {"opened", "reopened"}:
     sys.exit(0)
 
-id = event['pull_request']['node_id']
+id = event.get('pull_request', None)
```

Comment:
Would something like this work:

```python3
specific_event = event.get('pull_request', event.get('issue'))
id = specific_event.get('node_id')
```

then we might be able to do away with the if-statement assignments below.